src/async_fuse/fuse/de.rs

```diff
@@ -161,6 +170,7 @@ impl<'b> Deserializer<'b> {
     }
 
     /// Fetch remaining bytes and transmute to a slice of target instances
+    #[allow(dead_code)]
```

Comment:
Is this function still useful? or are we going to use it later?

Reply:
Yes, this function is used on 7.16 to parse `FUSE_BATCH_FORGET`, but unused on 7.8, so I have to mark it with `#[allow(dead_code)]`.

There is a similar problem in #423 .

Reply:
Shall we mark it with `#[cfg(...)]` instead of "dead_code"?

Reply:
We can do it, this function is not such ABI related, but it's certainly not used before 7.16. And the function `fetch_slice` above is not used in any version, perhaps we can remove it.